libs/ui/packages/native/package.json

```diff
@@ -40,6 +40,7 @@
     "build": "tsc -p tsconfig.prod.json && node scripts/postBuild",
     "prebuild:storybook": "pnpm -F ui-shared -F icons-ui -F crypto-icons-ui build",
     "build:storybook": "pnpm storybook build -c .storybook-web -o web-build",
+    "dev:storybook": "storybook dev -c .storybook-web -p 6006",
```

Comment:
The issue does not come from here. It's because in `build:storybook` we call `pnpm storybook build`
that means that pnpm will look at 'storybook' script and will run storybook dev instead of storybook build. you can see it with the error you have in terminal:
`storybook dev  -c .storybook-web -p 6006 "build" "-c" ".storybook-web" "-o" "web-build"`
If you remove pnpm it will work:
`"build:storybook": "storybook build -c .storybook-web -o web-build",`